.changeset/modern-lights-watch.md

```diff
@@ -0,0 +1,22 @@
+---
+"fluid-framework": minor
+"@fluidframework/tree": minor
+---
+---
+"section": tree
+---
+
+Improved typing of `TreeBeta.clone`
+
+The return type of `TreeBeta.clone` now matches the input type.
+Existing usages which were supplying the generic parameter for the input/output type should remove the explicit generic parameter and allow it to be inferred from the input.
```

Comment:
FWIW, we could make this a non-breaking change by keeping the old signature around as a second overload. However, since this is a beta API, it seems like it'd be best to just have any consumers be forced to fix it since it's so easy to fix and makes their code immediately nicer.